### PR TITLE
Add missing CSS classes to the Event Categories label

### PR DESCRIPTION
**Pull request by andrasguseo**
_Ref:_ [TEC-3478](https://moderntribe.atlassian.net/browse/TEC-3478)

Location where the classes are added:
https://dl.dropboxusercontent.com/s/3n29fbvovfl2phi/shot_200522_180144.jpg

### CODE REVIEW

**Comment by lucatume** [comment]
@andrasguseo could you update the description of this PR to provide some context for this change? Some screenshots?

**Comment by lucatume** [comment]
This PR is on hold while its ticket finds a home in the next release.

File: src/admin-views/admin-update-message.php [hunk 1]
```diff
@@ -6,7 +6,11 @@
 
 ?>
 
-<p class="tribe-welcome-message"><?php echo esc_html( sprintf( __( 'You are running Version %s and deserve a hug :-)', 'the-events-calendar' ), Tribe__Events__Main::VERSION ) ); ?></p>
+
+<p class="tribe-update-message"><?php
```
Comment (tr1b0t) [inline src/admin-views/admin-update-message.php:10]: :no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: 9456cc28da099eed3f9c7c00d8d4c804</sub>

File: src/admin-views/admin-update-message.php [hunk 2]
```diff
@@ -6,7 +6,11 @@
...
+	/* translators: the placeholder will contain the semantic version of The Events Calenda plugin, e.g. 1.2.3 */
+	echo esc_html( sprintf( __( 'You are running Version %s and deserve a hug :-)', 'the-events-calendar' ), Tribe__Events__Main::VERSION ) );
+	?></p>
```
Comment (tr1b0t) [inline src/admin-views/admin-update-message.php:13]: * :no_entry_sign: Line indented incorrectly; expected 0 tabs, found 1
* :no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: c142b332cc2f5518eee16f01adf451b5</sub>

File: src/views/blocks/featured-image.php [hunk 3]
```diff
@@ -7,12 +7,12 @@
...
  *
  * @version 4.7
  *
  */
 
 $event_id = $this->get( 'post_id' );
-?>
-<?php echo tribe_event_featured_image( $event_id, 'full', false );
\ No newline at end of file
+
```
Comment (tr1b0t) [inline src/views/blocks/featured-image.php:17]: * :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'tribe_event_featured_image'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 68b46e3aafc8ba116032007f799e8baa</sub>

File: src/Tribe/Main.php [hunk 4]
```diff
@@ -35,7 +35,7 @@ class Tribe__Events__Main {
 		const VENUE_POST_TYPE     = 'tribe_venue';
 		const ORGANIZER_POST_TYPE = 'tribe_organizer';
 
-		const VERSION             = '5.1.6';
+		const VERSION             = '5.1.7';
```
Comment (tr1b0t) [inline src/Tribe/Main.php:38]: :warning: Equals sign not aligned correctly; expected 1 space but found 13 spaces

<sub>hash: 1eb125f5345e78d2763bd5b0802a3232</sub>

**Review by lucatume — APPROVED** [review]

**Pull request merged**